tests/unit/controllers/content-test.js

```diff
@@ -1,12 +1,98 @@
+        copyright_holders: []
+    });
+    ctrl.notifyPropertyChange('model.license');
+    assert.equal(ctrl.get('fullLicenseText'), 'On , for ');
+
+    preprint.set('licenseRecord',  {
+        year: '{{year}}',
+        copyright_holders: ['{{copyrightHolders}}']
+    });
+    ctrl.notifyPropertyChange('model.license');
+    assert.equal(ctrl.get('fullLicenseText'), 'On {{year}}, for {{copyrightHolders}}');
+});
+
+test('useShortenedDescription computed property', function(assert) {
+    let ctrl = this.subject();
+    let node = FactoryGuy.make('node');
+    ctrl.set('node', node);
+    Ember.run(() => {
+        node.set('description', 'string'.repeat(100).slice(0, 351));
+        ctrl.notifyPropertyChange('node.description');
+        assert.ok(ctrl.get('useShortenedDescription'));
+
+        node.set('description', 'string'.repeat(100).slice(0, 350));
+        ctrl.notifyPropertyChange('node.description');
+        assert.ok(!ctrl.get('useShortenedDescription'));
+
+        node.set('description', 'string'.repeat(100).slice(0, 349));
+        ctrl.notifyPropertyChange('node.description');
+        assert.ok(!ctrl.get('useShortenedDescription'));
+    });
+});
+
+test('description computed property', function(assert) {
+    Ember.run(() => {
+        let ctrl = this.subject();
+        let node = FactoryGuy.make('node');
+        ctrl.set('node', node);
+        ctrl.set('expandedAbstract', false);
+
+        //Test cut at 350 characters
+        let description = 'string string string string string string string string string string string string string string string string string string string string string string string string string string string string string string string string string string string string string string string string string string string string string string string string string string string str';
+        let notExpanded = description.slice(0, 350);
+        node.set('description', description);
+        ctrl.notifyPropertyChange('description');
+        assert.equal(ctrl.get('description'), notExpanded.trim());
+
+        //Test cut at less than 350 characters to not cut in middle of word
+        description = 'string stringstringstringstringstringstringstringstringstringstringstringstringstringstringstringstringstringstringstringstringstringstringstringstringstringstringstringstringstringstringstringstringstringstringstringstringstringstringstringstringstringstringstringstringstringstringstringstringstringstringstringstringstringstringstringstringstringstringstring';
+        notExpanded = 'string ';
+        node.set('description', description);
+        ctrl.notifyPropertyChange('description');
+        assert.equal(ctrl.get('description'), notExpanded.trim());
+
+        //Test less than 350 doesn't use description (uses node.description)
+
+        ctrl.set('expandedAbstract', true);
+        // expandedAbstract logic has been moved to the template, making the expanded tests unnescessary here.
```

Comment:
Per this note: does this mean that any code after the last assertion can be removed?